MvvmCross/Core/Core/Platform/MvxStringToTypeParser.cs

```diff
@@ -1,4 +1,4 @@
-﻿// MvxStringToTypeParser.cs
+// MvxStringToTypeParser.cs
```

Comment:
The parameters passed to ```ShowViewModel``` are  deserialized in this class. Need ```TryParse``` with ```CultureInfo.InvariantCulture```. I added ```CultureInfo.InvariantCulture```. But with style changed...